app/views/layouts/blacklight/base.html.erb

```diff
@@ -32,6 +32,17 @@
 
   <%= render partial: 'shared/flash_msg', layout: 'shared/flash_messages' %>
 
+  <% if @search_context %>
```

Comment:
So, I'm curious as to why you had to move the search context bit and the previous next doc bit here. Seems unrelated to the format work, but I'm missing something, right?

Reply:
after adding thumbnail, pagination widgets and start over did not look good in the content section so I moved them out here so they expand over both sections.

Reply:
that makes sense! nice work, thanks.